Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:2.7.12
```

Comment:
We should use consul-template-base and install python with apt. It's a lot simpler.